server/routes/blogs.js

```diff
@@ -0,0 +1,77 @@
+const router = require('express').Router();
+const authorization = require('../middleware/authorization');
+const pool = require('../db');
+
+router.post('/add', authorization, async (req,res)=>{
+    const {
+      user_name,
+      question,
+      blog_text,
+      date
+    } = req.body;
+
+
+    const blog = await pool.query("INSERT INTO blogs (user_id, user_name, question, blog_text, blog_date) VALUES($1, $2, $3, $4, $5) RETURNING *", [
```

Comment:
For the time you wrote this, I feel sorry for writing this comment, but here SQL injection is VERY possible